app/presenters/two_factor_authentication/phone_selection_presenter.rb

```diff
@@ -2,18 +2,28 @@ module TwoFactorAuthentication
   class PhoneSelectionPresenter < SelectionPresenter
     def type
       if MfaContext.new(configuration&.user).phone_configurations.many?
-        "#{super}:#{configuration.id}"
```

Comment:
As I mentioned in my commit message, it's for two reasons: consistency, and because colons are not valid CSS id selectors and causes Capybara to fail when looking for them. See the feature specs.